partitioned-table.md

```diff
@@ -579,7 +579,21 @@ You can see that the inserted record `(NULL, 'mothra')` falls into the same part
 
 ## Partition management
 
-You can add, drop, merge, split, redefine partitions by using `ALTER TABLE` statements.
+For `LIST` and `RANGE` partitioned tables, you can add and drop partitions using the `ALTER TABLE <table name> ADD PARTITION (<partition specification>)` or `ALTER TABLE <table name> DROP PARTITION <list of partitions>` statement.
+
+For `HASH` partitioned tables, `COALESCE PARTITION` and `ADD PARTITION` are not yet supported.
+
+`REORGANIZE PARTITION` is not yet supported.
+
+> **Warning:**
+>
+> - `EXCHANGE PARTITION` is an experimental feature. It is not recommended to use it in a production environment. To enable it, set the `tidb_enable_exchange_partition` system variable to `ON`.
```

Comment:
+ Warning notes are used to provide additional information to the content above. There seems no mention of `EXCHANGE PARTITION` before this warning note.
+ The warning note only serves as a reminder or alert, and should be concise enough. One or two sentences should be enough. Detailed explanation or even example are not recommended. Could you consider putting some of the information out of the warning note?